governance-exercises/Le Bateleur - Interpretation Exercise 01/GOV 12 Rewrite.md

```diff
@@ -0,0 +1,56 @@
+
+My rewrite assumes that associated provisions in the rulebooks (i.e., [Atlas 2.12)](https://mips.makerdao.com/mips/details/MIP101#2-12-scope-bootstrapping-gov12) would have to be interpreted or re-written so as to be reconciled with the revised GOV 12. The revised GOV 12 could complement [MIP24](https://mips.makerdao.com/mips/details/MIP24#sentence-summary), an emergency voting protocol that may be initiated by any community member and manages urgent changes to the protocol. In contrast, the revised GOV 12 is a temporary measure authorizing only Governance Facilitators, under certain circumstances, to expedite revisions to the Atlas and Scopes Artifacts.  
+
+My rewrite also assumes that the Governance Facilitators interpreting GOV 12 are comfortable applying fuzzy legal terms of art. (There would be distinct benefits: the civil law tradition is a rich one going back centuries. Arguably, it is as close as we can get to a stable, predictable foundation for drafting and interpreting governance logic.) 
+
+---
+
+### 12: Scope Bootstrapping
+
+#### 12.1 Bootstrapping Governance Votes
+
+Before the Endgame State, time-sensitive situations may arise that pose a substantial threat to the functioning of Maker Governance. Under the conditions and limitations set out below, Governance Facilitators may propose a Governance Poll to edit the Atlas and Scope Artifacts to address such substantial threats. This authority is permanently eliminated upon the launch of NewChain.
+
+#### 12.2 Finding of "substantial threat"
+
+A substantial threat to the functioning of Maker Governance is one that extends beyond causing mere inconvenience, annoyance or dissatisfaction. To be "substantial," the threat must be of both considerable *imminence* and *magnitude*. Therefore, a threat that is of considerable magnitude but is not likely to present in the near-term could not be characterized as "substantial."
+
+To determine whether a substantial threat exists to the functioning of Maker Governance such that GOV 12 may be triggered, the Governance Facilitator must engage in a balancing inquiry. This inquiry explicitly weighs two interests: the imminence and magnitude of the threat, against the need to uphold a decentralized governance equilibrium pursuant to customary governance protocols. 
```

Comment:
I get the feeling that the term 'balancing inquiry' is something that exists from elsewhere. Given potential unfamiliarity with the term, might be worth explicitly providing or linking to a definition.